backend/api/v1/routes/query.py

```
@@ -0,0 +1,196 @@
+            execution_time_ms=execution_time_ms,
+            tables_used=tables_used
+        )
+        
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.post("/execute", response_model=QueryResponse)
+async def execute_sql(
+    request: QueryRequest,
+    x_tenant_id: str = Header(..., description="고객사 ID")
```

Ruff (0.13.1)

Unused function argument: `x_tenant_id`

(ARG001)